source/administration/batch-framework-job-expire.rst

```diff
@@ -106,7 +106,7 @@ Expire Batch Job Reference
    * - ``rules.[n].createdBefore``
      - *Optional*
 
-       Specify an ISO-8601 timestamp for filtering objects.
+       Specify an RFC3339 date and time for filtering objects.
```

Comment:
If you want to be super cool you can do:

```suggestion
       Specify an :rfc:`RFC3339 <3339>` date and time for filtering objects.
```

https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-rfc